artiq/devices/novatech409B/novatech409B.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2015 Joe Britton
+
+import serial
+import sys
+import time
+import math
+import platform
+import logging
+import inspect
+
+
+class Novatech409B:
+    """controller for Novatech 409B 4-channel DDS
+
+    This class is an interface with the Novatech Model 409B 4-
+    channel DDS box. The interface is a serial interface.
+
+    :param int comport: COM port number on Windows
+    :param int debug: debug level
+    """
+    # NOTE: the in-line documentation is in Sphinx format
+    # http://sphinx-doc.org/domains.html
+    def __init__(self, comport=1, debug=1, simulate_hw=False):
+        # some private members
+        self.__comport = comport
+        self.__debug = debug
+        self.__className = "Novatech409B"
+        self.simulate_hw = simulate_hw  # true if disconnected from hw
+        self.serial_rw_delay = 0.001  # is time between reads and writes
+
+        # setup logging
```

Comment:
A line comment here refers to the line just above the comment, `# setup logging` in this case. What Yann wrote.